### ISSUE TITLE
discussion needed if we want to stick to openMINDS_MRI

### ISSUE THREAD

**Issue by lzehl** [openMetadataInitiative/openMINDS_MRI#2]
I'd like to discuss if it wouldn't make sense to have a repo where we can put other neuroimaging techniques as well over time, instead of splitting them all up. 

If we go with MRI we need to rename the techniques. MRI should at least include all magnetic resonance imaging techniques (all functional and all structural ones).

@openMetadataInitiative/openminds-developers (+community) this is up for discussion :wink:

**Comment by Peyman-N** [comment]
I agree there is a big similarity between all of the neuroimaging (specially in subject preparation aspect) but the similarity ends there, for example MEG (Magnetoencephalography) thought it is categorized as neuroimaging is more close to EEG than MRI. So putting all of them together would introduce to much inhemoginity. 
But I agree we should minimize the number of repos, so I would categorize them into three repos:

1. MRI (including all the functional and structural images) 
2. CT, PET, SPECT (I still have no name for this category, I wanted to call them tomography but I think it is too general) 
3. MEG

in future we can add following repose:
1. Optical imaging (Near Infrared Spectroscopy (NIRS), Event-related optical signal (EROS))
2. Ultrasound based ones

Currently with pull request #1  we support most of the structural MR images and functional images (ASL and BOLD), saying that we can still improve it and add more modalities specially in Perfusion based MR images and Angiography. 

Anyway all of this is still up for discussion.

**Comment by lzehl** [comment]
This issue got solved. We introduced openMINDS_neuroimaging
